src/libcore/convert.rs

```diff
@@ -48,7 +48,24 @@
 
 #![stable(feature = "rust1", since = "1.0.0")]
 
-use str::FromStr;
+use fmt;
+
+/// An uninhabited type used as the error type for implementations of fallible
```

Comment:
I think that adding "uninhabited" here might confuse new users. The latter paragraph explains better so I think we can just omit it?